pkgs/by-name/ks/ks/package.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "ks";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "loteoo";
+    repo = "ks";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-jGo0u0wiwOc2n8x0rvDIg1suu6vJQ5UCfslYD5vUlyI=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ${finalAttrs.pname} $out/bin/
+  '';
+
+  meta = {
+    mainProgram = "ks";
+    homepage = "https://github.com/loteoo/ks";
+    description = "Command-line secrets manager powered by macOS keychains";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ ivankovnatsky ];
+    platform = lib.platforms.darwin;
```

Comment:
CI meta check is failing because platform is not a valid meta attribute it should be `platforms`

Reply:
Updated, can you point please me how to check that, so I could do it next time first.

Reply:
You can check the ofborg log for the failing job by following the link from the ❌ job on your previous [commit](https://github.com/NixOS/nixpkgs/commit/4b56de9450627d3daf25343fe7724d0ee36e1eb9): https://gist.github.com/GrahamcOfBorg/42df16c9ec93f97612757c02d941c9c9

Reply:
![image](https://github.com/NixOS/nixpkgs/assets/12160/c31557d5-6059-4802-8cf7-846084f5780f)

it already started re-eval due to new changes being pushed, but one of checks is metadata check, and it was failing